### ISSUE TITLE
[announcement] New CLAs, please read

### ISSUE THREAD

**Issue by latkin** [Microsoft/visualfsharp#363]
***TL;DR*** - *code contributors need to sign new CLAs from https://cla.microsoft.com/.* ***Employer signoff is no longer required.***

Visual F# was previously under the legal auspices of Microsoft Open Technologies (note the copyright header on all of the source code), but has now been moved back under the aegis of normal Microsoft Corporation.

These are separate legal entities and require distinct CLAs.  So if you are a long-time contributor and have only submitted a Microsoft Open Tech CLA (via https://cla.msopentech.com/), you will need to submit a new one, from https://cla.microsoft.com/.

The good news is that the employer signoff requirement from the old CLA process has been eliminated. Submission of the new CLA only takes a couple of minutes.

![image](https://cloud.githubusercontent.com/assets/5943573/6947197/6e941e02-d85a-11e4-82e7-1b8b324499f5.png)

**Comment by forki** [comment]
Thanks @latkin, new CLA process is MUCH better than the last one. Kudos.

**Comment by latkin** [comment]
Well @KevinRansom and I deserve no credit for this, but I will pass along the sentiment :wink:

**Comment by dsyme** [comment]
Cool, much simpler

**Comment by simonhdickson** [comment]
10/10 would sign again

**Comment by rojepp** [comment]
+1, very smooth. Does this new CLA apply to all MS open source projects or just this one?

**Comment by latkin** [comment]
There are (currently) 3 buckets, and you only need to sign a CLA once for each bucket.  The legalese and submittal process is almost identical in each.

- **Microsoft Corp** (https://cla.microsoft.com/ - contains Visual F#, most of the stuff at https://github.com/microsoft/)
- **Microsoft Open Technologies** (https://cla.msopentech.com/ - previously our home, don't recall other specific projects)
- **.NET Foundation** (https://cla.dotnetfoundation.org/ - contains Roslyn, .NETCore, stuff under https://github.com/dotnet/)

**Comment by bryanhunter** [comment]
Hurray!!! I know firsthand how tricky the previous "employer sign-off" requirement was. My company spent thousands trying to figure a legal path through it, but our counsel concluded "signing off" on the previous Microsoft Open Tech CLA would have put the company in breach on each of its consulting agreements. Ouch!

Thankful to all that made this happen!

**Comment by enricosada** [comment]
done! easy
